crepecake/src/main/java/com/google/cloud/tools/crepecake/blob/BlobWriter.java

```diff
@@ -19,6 +19,10 @@
 import java.io.IOException;
 import java.io.OutputStream;
 
+/**
+ * Function that writes the contents of a BLOB. This functions should write the same BLOB regardless
+ * of the number of times it is called.
```

Comment:
Isn't the point of this that it can only be called once?

Reply:
`FileBlob`, `StringBlob`, and `WriterBlob` should all be able to be written out more than once. However, `InputStreamBlob` is tied to a read-once InputStream, so it can only be written out once.